hpy/tools/autogen/public_api.h

```diff
@@ -154,7 +154,7 @@ HPy_ID(97)
 double HPyFloat_AsDouble(HPyContext *ctx, HPy h);
 
 HPy_ID(98)
-HPy HPyBool_FromLong(HPyContext *ctx, long v);
+HPy HPyBool_FromBool(HPyContext *ctx, bool v);
```

Comment:
I wonder if we should have some `HPy_bool_t` to also remove the compiler-specific `bool`. Not sure how far we want to take this -- do we want to remove all sources of potential compile-specifics from ABI?

Reply:
> the compiler-specific `bool`

As far as I know, the Boolean type (i.e. `_Bool` or `bool`) is part of the C standard since C99 (see, e.g.,  https://en.cppreference.com/w/c/types). That's why I thought that we could just have it in the ABI, in particular because `true` and `false` are now also defined to expand to `1` and `0`, respectively.

What we should definitively do is to explicitly make C99 the minimum required standard for HPy (right now, this is just an implicit requirement).

Reply:
I agree that we should specify C99 as a minimum required standard.